app/src/main/java/ch/epfl/sweng/erpa/model/UserAuth.java

```diff
@@ -0,0 +1,51 @@
+package ch.epfl.sweng.erpa.model;
+
+import java.util.Objects;
+
+public class UserAuth {
+
+    private String uid;
+    private String accessToken;
+
+    public UserAuth(String uid, String accessToken) {
+        this.uid = uid.toUpperCase();
```

Comment:
Where do you add the “user/” prefix (as discussed)?
Can you please add an “user/” prefix? Else, make sure the uid has a “user/” prefix (else fail with an exception)

Reply:
I add it at the sign up I don't need to add it each time I retrieve it